benchto-service/src/main/java/com/teradata/benchto/service/BenchmarkService.java

```diff
@@ -126,7 +127,7 @@ public void startExecution(String uniqueName, String benchmarkSequenceId, String
     @Retryable(value = {TransientDataAccessException.class, DataIntegrityViolationException.class}, maxAttempts = 1)
```

Comment:
Can you do a little investigation? To check why it is set to 1. Because I think it would be easier to remove this annotation at all. Maybe I don't know how this `@Retryable` works.